src/Components/layout/header.js

```diff
@@ -0,0 +1,67 @@
+import { NavLink } from 'react-router-dom';
+import Logo from '../../assets/photos/planet.png';
+
+const Header = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/Rockets',
+      text: 'Rockets',
+    },
+    {
+      id: 2,
+      path: '/Missions',
+      text: 'Missions',
+    },
+    {
+      id: 3,
+      path: '/Profile',
+      text: 'Profile',
+    },
+  ];
+
+  return (
+    <>
+      <nav className=" grid grid-cols-5 p-3">
+        <div className=" col-span-2 m-auto flex justify-center w-full">
+
+          <div className="flex w-full justify-around">
+            <NavLink
+              to={links[0].path}
+              data-testid="link-item"
+              className=""
+            >
+              <img src={Logo} alt="logo" />
+            </NavLink>
+            <NavLink
+              to={links[0].path}
+              data-testid="link-item"
+              className=""
+            >
+              <h1 className="md:text-2xl hover:text-blue-600 hidden sm:block text-xl font-bold font-italics">Space Travelers Hub</h1>
+
+            </NavLink>
+          </div>
+        </div>
+        <div className="col-span-3 flex justify-center">
+          <ul className="grid grid-cols-3 w-full">
+            {links.map((link) => (
+              <li key={link.id} className="font-normal text-base hover:text-blue-600">
+                {' '}
+                <NavLink
+                  to={link.path}
+                  data-testid="link-item"
+                  style={({ isActive }) => ({ opacity: isActive ? 1 : 0.5, textDecoration: isActive ? 'underline' : 'none' })}
+                >
+                  {link.text}
+                </NavLink>
+              </li>
+            ))}
+          </ul>
+        </div>
```

Comment:
- [ ] @team great work on your navbar however the positioning of the mission rockets and profile links si wrong also the color of mission and rockets text differe from design given, check the comparison below:

Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/209115909-3d6a7408-7b6e-47c7-8c57-91fd83dd979d.png)| ![image](https://user-images.githubusercontent.com/61408860/209116023-cd52c489-8755-401d-ba8c-598370ba2fc4.png)